autotest/ogr/ogr_mssqlspatial.py

```diff
@@ -265,6 +266,101 @@ def ogr_mssqlspatial_test_ogrsf():
 
     return 'success'
 
+###############################################################################
+# Verify features can be created in an existing table that includes a geometry
+# column but is not registered in the "geometry_columns" table.
+
+
+def ogr_mssqlspatial_create_feature_in_unregistered_table():
+    if gdaltest.mssqlspatial_ds is None:
+        return 'skip'
+
+    # Create a feature that specifies a spatial-reference system
+    spatial_reference = osr.SpatialReference()
+    spatial_reference.ImportFromEPSG(4326)
+
+    feature = ogr.Feature(ogr.FeatureDefn('Unregistered'))
+    feature.SetGeometryDirectly(ogr.CreateGeometryFromWkt(
+        'GEOMETRYCOLLECTION ('
```

Comment:
any reason not to put a simpler geometry like POINT(1 2) ?

Reply:
No, except to generally stress the driver as much as possible.

I can change this to a simpler type if you prefer.

Reply:
Yes, geometry test stressing could be a dedicated test step